include/vcpkg/base/json.h

```diff
@@ -210,10 +210,10 @@ namespace vcpkg::Json
     public:
         // these are here for better diagnostics
         Object() = default;
-        Object(Object const&) = default;
-        Object(Object&&) = default;
-        Object& operator=(Object const&) = default;
-        Object& operator=(Object&&) = default;
+        Object(const Object&) = default;
+        Object(Object&&) noexcept = default;
+        Object& operator=(const Object&) = default;
+        Object& operator=(Object&&) noexcept = default;
         ~Object() = default;
 
```

Comment:
I think they should all just be deleted. I don't know what they meant by "for better diagnostics" but since they all =default they have no effect.